lib/plugins.ts

```diff
@@ -98,7 +98,7 @@ export interface ProtractorPlugin {
    *     will *not* wait before executing the next test, however.  If the promise
    *     is rejected, a failed assertion is added to the test results.
    */
-  postTest?: (passed: boolean, testInfo: any) => q.Promise<any>;
+  postTest?(passed: boolean, testInfo: any): void|q.Promise<void>;
```

Comment:
These can actually return any kind of promise.  How about `void|q.IPromise`?  Also, update the comment to have a return type `{Promise=}`.